test/functional/shada/marks_spec.lua

```diff
@@ -104,7 +104,7 @@ describe('ShaDa support code', function()
 
   it('is able to dump and read back mark " from a closed tab', function()
     nvim_command('edit ' .. testfilename)
-    nvim_command('edit ' .. testfilename_2)
+    nvim_command('tabedit ' .. testfilename_2)
     nvim_command('2')
     nvim_command('q!')
     nvim_command('qall')
```

Comment:
From our discussion:

- This `nvim_command()` request is being sent to a dead session. In general, the `helpers` methods silently fail if the session is dead. `eq(2, eval('1 + 1'))` is an explicit way to detect a dead session. But we should change lua-client and/or the lua test helpers to be more vocal about this situation.
- On the debian infrastructure the tests died here, and that's how this was found. We don't know why it behaves differently on that machine.